crates/starknet/src/starknet/mod.rs

```diff
@@ -467,6 +576,23 @@ impl Starknet {
         add_invoke_transaction::add_invoke_transcation_v1(self, invoke_transaction)
     }
 
+    // save starknet transactions to file
+    pub fn dump_transactions(&self) {
```

Comment:
```suggestion
    pub fn dump_transactions(&self) -> Result<()> {
```
Exoects have to go :(

Reply:
Fixed in 27ec8cc63261fb9430c873a0adda2ceebf38f005